e2e/sites.test.ts

```diff
@@ -65,7 +68,7 @@ test.describe( 'Servers', () => {
 		const response = await new Promise< http.IncomingMessage >( ( resolve, reject ) => {
 			http.get( `http://${ frontendUrl }`, resolve ).on( 'error', reject );
 		} );
-		expect( response.statusCode ).toBe( 200 );
+		expect( [ 200, 302 ] ).toContain( response.statusCode );
```

Comment:
Looks weird. So sometimes when we run test we get 200 and sometimes 302?

Reply:
In my testing I received only 302 which is  about the content being temporarily moved. I added both here for robustness, as I think the WP frontend can respond either one.